File: java/Task.java
/**
 * Task class representing a data processing task
 */
public class Task {
    private int id;
    private String data;
    private long timestamp;
    
    public Task(int id, String data) {
        this.id = id;
        this.data = data;
        this.timestamp = System.currentTimeMillis();
    }
    
    public int getId() {
        return id;
    }
    
    public String getData() {
        return data;
    }
    
    public long getTimestamp() {
        return timestamp;
    }
    
    @Override
    public String toString() {
        return "Task{id=" + id + ", data='" + data + "', timestamp=" + timestamp + "}";
    }
}

File: go/task.go
package main

import (
	"fmt"
	"time"
)

// Task represents a data processing task
type Task struct {
	ID        int
	Data      string
	Timestamp time.Time
}

// NewTask creates a new task with the given ID and data
func NewTask(id int, data string) *Task {
	return &Task{
		ID:        id,
		Data:      data,
		Timestamp: time.Now(),
	}
}

// String returns a string representation of the task
func (t *Task) String() string {
	return fmt.Sprintf("Task{id=%d, data='%s', timestamp=%s}", t.ID, t.Data, t.Timestamp.Format("2006-01-02 15:04:05"))
}
